### PR TITLE
New Zigbee Plugs added

### PR DESCRIPTION
**Pull request by dhanukapulkit**

### CODE REVIEW

**Comment by Koenkk** [comment]
It looks that all devices you added are already supported, I guess something went wrong?

File: src/devices/ledvance.ts [hunk 1]
```diff
@@ -38,6 +42,36 @@ const definitions: Definition[] = [
         description: 'Panel TW LED 625 UGR19',
         extend: [ledvanceLight({colorTemp: {range: [153, 370]}, color: true})],
     },
+    {
+        zigbeeModel: ['PLUG EU EM T'],
+        model: '4099854295232',
+        vendor: 'LEDVANCE',
+        description: 'SMART+ Indoor Plug EU with Energy meter ',
+        extend: [onOff(), electricityMeter()],
+        configure: async (device, coordinatorEndpoint, logger) => {
```
Comment (Koenkk) [inline src/devices/ledvance.ts:51]: Could you also try removing the `configure` and `exposes`?

**Review by dhanukapulkit — COMMENTED** [review]
removed configure and exposes

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**